src/main/c/ssl.c

```diff
@@ -719,8 +719,8 @@ TCN_IMPLEMENT_CALL(jint, SSL, initialize)(TCN_STDARGS, jstring engine)
      * low entropy seed.
      */
     SSL_rand_seed(NULL);
-    /* For SSL_get_app_data2() at request time */
-    SSL_init_app_data2_idx();
+    /* For SSL_get_app_data2() and SSL_get_app_data3() at request time */
```

Comment:
actually data2 is used by tcnative and data3 I just added. Let me add a comment in the code